cmd/mayactl/app/command/volume_info.go

```diff
@@ -323,5 +323,6 @@ Replica Count :   {{.ReplicaCount}}
 		}
 		w.Flush()
 	}
+	fmt.Println("Unsupported Volume Type")
```

Comment:
Are we missing the else statement ? Won't this message be printed in all cases ?

Reply:
fixed it!  my silly mistake